mlflow/sklearn/utils.py

```diff
@@ -4,6 +4,7 @@
 import logging
 from numbers import Number
 import numpy as np
+from sklearn.metrics import ConfusionMatrixDisplay
```

Comment:
Can we import this in `_get_classifier_artifacts`?

Reply:
If we import `ConfusionMatrixDisplay` here, users who don't need to use `mlflow.sklearn` need to install `sklearn`.